apps/examples/webclient/webclient_main.c

```diff
@@ -364,10 +404,26 @@ int webclient_main(int argc, char *argv[])
 	status = pthread_attr_setschedpolicy(&attr, WEBCLIENT_SCHED_POLICY);
 	status = pthread_attr_setstacksize(&attr, WEBCLIENT_STACK_SIZE);
 
-	arg.argc = argc;
-	arg.argv = argv;
+	input = (struct webclient_input *)malloc(sizeof(struct webclient_input));
+	if (!input) {
+		printf(" malloc fail\n");
+		return 0;
+	}
+	input->argv = (char **)malloc(sizeof(char *) * argc);
+	if (!input->argv) {
+		printf(" malloc argv fail\n");
+		return 0;
+	}
+
+	input->argc = argc;
+	int i = 0;
+	for (; i < argc; i++) {
+		input->argv[i] = (char *)malloc(sizeof(char) * (strlen(argv[i]) + 1));
+		strcpy(input->argv[i], argv[i]);
+	}
+
+	status = pthread_create(&tid, &attr, webclient_cb, input);
```

Comment:
In failure case, you should add free allocated memory.
```
if (status < 0) {
  for (i=0; i < argc; i++) {
    free(input->argv[i]);
  }
  free(input->argv);
  free(input);
  xxx
}
```

Reply:
done